### PR TITLE
Update ceph to v16.2.10 to address CVE-2022-0670

### PR DESCRIPTION
**Pull request by Camelron**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Update ceph to v16.2.10

###### Change Log  

- Update ceph to v16.2.10
- Remove snappy patch, since it is addressed in v16.2.7

###### Does this affect the toolchain?  

NO

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2022-0670

###### Test Methodology

- https://dev.azure.com/mariner-org/mariner/_build/results?buildId=223465&view=results

### CODE REVIEW

File: SPECS/ceph/ceph.signatures.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "ceph-16.2.5.tar.gz": "b8206916fa7993e0394a0c31a09564b866fd13af6968aaecdea75ae51a6c46de"
+  "ceph-16.2.10.tar.gz": "df8dbe9d45779825fb4097d94a72847e7405089c0999541e62779f2705bf02d5"
```
Comment (PawelWMS) [inline SPECS/ceph/ceph.signatures.json:3]: Why 16.2.10 and not 17.2.2? [NIST](https://nvd.nist.gov/vuln/detail/CVE-2022-0670#range-8182707) seems to suggest that's the first version with this CVE patched.

**Review by mfrw — COMMENTED** [review]
For what it's worth; it seems, we might have a build-break.

I got an error:

`install: cannot stat 'sudoers.d/ceph-osd-smartctl': No such file or directory"`

Maybe we should double check.

File: SPECS/ceph/ceph.signatures.json (hunk 1 above)
Comment (Camelron) [reply SPECS/ceph/ceph.signatures.json:3]: For our major version (Pacific, 16.X.X), 16.2.10 is the version with the fix. See here https://docs.ceph.com/en/latest/security/CVE-2022-0670/

Comment (PawelWMS) [reply SPECS/ceph/ceph.signatures.json:3]: Amazing, thank you! In that case please also send NIST an e-mail, so they'd update their database to include 16.2.10 as a fixed version. It's the polite thing to do and also without it our tooling will keep complaining about `ceph`.;)

**Review by PawelWMS — APPROVED** [review]
Just one request to send an e-mail to NIST but that's outside the scope of this PR.

**Review by mfrw — CHANGES_REQUESTED** [review]
Blocking the PR:
I've got one more build break:
`cp: cannot stat 'monitoring/grafana/README.md': No such file or directory`

**Review by mfrw — APPROVED** [review]
LGTM :)
Thanks for addressing the comments.

File: SPECS/ceph/ceph.signatures.json (hunk 1 above)
Comment (Camelron) [reply SPECS/ceph/ceph.signatures.json:3]: email is sent

**Pull request merged**